### PR TITLE
Add the Giant Swarm Developer Platform

### PR DESCRIPTION
**Pull request by teemow**
### What does this PR do?

 * Added a new section "Developer Platform"
 * Moved the app platform under the developer platform
 * Moved the managed apps into connectivity, observability and security
 * Added service mesh and ingress to connectivity
 * Fixed all old links to the app platform
 * Added symlinks from the old urls to new ones
 * Replaced some non-relational links that contained "docs.giantswarm.io"
 * Tested locally

### What does it look like?

screen_2022-11-14-13-51-03
screen_2022-11-14-13-51-07
screen_2022-11-14-13-51-11
screen_2022-11-14-13-51-14
screen_2022-11-14-13-56-48

### Any background context you can provide?

(Please link public issues or summarize if not public.)

### Have you maintained the front matter?

(Please bump the last_review_date in case this qualifies as a review for an entire page. Provide user_questions which should be answered in the page. Provide a meaningful description.)

### CODE REVIEW

**Comment by teemow** [comment]
I know the content isn't perfect yet. But at least the new structure makes sense imo. Let's iterate from here :pray:

File: src/content/developer-platform/observability/_index.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
 ---
-linkTitle: Observability stack
-title: Managed observability stack (monitoring, logging, service mesh)
-description: Overview of the Giant Swarm managed observability stack of apps, which includes monitoring, logging, and service mesh
-weight: 10
+linkTitle: Observability
+title: Developer Platform Observability Features (monitoring, logging, tracing)
```
Comment (TheoBrigitte) [inline src/content/developer-platform/observability/_index.md:3]:
```suggestion
title: Developer Platform Observability Features
```

File: src/content/developer-platform/observability/_index.md [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+description: Overview of the observability related platform features, which includes monitoring, logging, and tracing
```
Comment (TheoBrigitte) [inline src/content/developer-platform/observability/_index.md:4]:
```suggestion
description: Overview of the observability related platform features to help you identify, understand and improve your system better.
```

**Review by TheoBrigitte — CHANGES_REQUESTED** [review]
Just changing some wording, but this might need more refactoring.

File: src/content/developer-platform/observability/_index.md (hunk 2 above)
Comment (teemow) [inline src/content/developer-platform/observability/_index.md:4]:
```suggestion
description: Overview of the observability related platform features to help you operate and improve your platform and applications.
```

@TheoBrigitte good idea. I would go with this :arrow_up:

**Pull request merged**